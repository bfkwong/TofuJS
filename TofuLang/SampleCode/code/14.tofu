s = [1, 2, 3, 4, 5];

for each item in s {
  print(item);
}

for each item in s {
  if (item > 3) is true then {
    print(100);
  } or else {
    print(item);
  }
}
