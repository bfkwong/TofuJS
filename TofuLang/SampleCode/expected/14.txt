1
2
3
4
5
1
2
3
100
100
